### PR TITLE
Fixes #21255 - Show correct envs for docker tags

### PR DESCRIPTION
**Pull request by jturel**
Only show the Lifecyle Environments for the docker tag we are looking at - not all of them!

### CODE REVIEW

**Comment by parthaa** [comment]
> @parthaa I could use some feedback on one point - is removal of uuid from the RABL doc valid? Looks to me like only the frontend piece is looking at it.

I am totally ok removing the uuid  column. I dont see any one using it with respect to the api. More over it doesnot make sense with the DockerMetaTag since that entity doesnot have the concept of UUID

File: app/models/katello/docker_meta_tag.rb [hunk 1]
```diff
@@ -49,6 +53,10 @@ def schema2_manifest
       schema2.try(:docker_manifest)
     end
 
+    def self.with_uuid(ids)
```
Comment (parthaa) [inline app/models/katello/docker_meta_tag.rb]: I am mixed about whether you should do something like
"alias_method ...." . Feel free to add

**Comment by parthaa** [comment]
LGTM APJ

**Comment by jlsherrill** [comment]
[test]

**Pull request merged**